lte/gateway/fabfile.py

```diff
@@ -617,32 +463,6 @@ def build_and_start_magma_trf(c, destroy_vm=False, provision_vm=False):
         _start_trfserver(c_trf)
 
 
-def _copy_out_c_execs_in_magma_vm(c_gw):
-    exec_paths = [
-        '/usr/local/bin/sessiond', '/usr/local/bin/mme',
-        '/usr/local/sbin/sctpd', '/usr/local/bin/connectiond',
-        '/usr/local/bin/liagentd',
-    ]
-    dest_path = '~/magma-packages/executables'
-    c_gw.run('mkdir -p ' + dest_path, warn=True)
-    for exec_path in exec_paths:
-        if not c_gw.run(f"test -e {exec_path}", warn=True).ok:
-            print(exec_path + " does not exist")
-            continue
-        c_gw.run('cp ' + exec_path + ' ' + dest_path, warn=True)
-
-
-def _build_magma(c_gw):
```

Comment:
I believe this is still referred to on line 275, i.e. in `_build_and_start_magma`